### PR TITLE
Added idle time support for gnome/wayland

### PR DESCRIPTION
**Pull request by filipkilibarda**
Pretty hacky as I don't know a standard way to determine user idle time. But I'm running GNOME/Wayland so this works for me. Maybe it could work for others.

### CODE REVIEW

**Comment by filipkilibarda** [comment]
#286

**Comment by slgobinath** [comment]
@filipkilibarda thank you very much for your contribution. I have no idea about wayland as I am using Linux Mint. I will test this feature in a VM and merge the changes if it works.

BTW could you please check if you can use `Utility.DESKTOP_ENVIRONMENT == 'gnome'` in place of the `is_gnome()` function.

**Comment by filipkilibarda** [comment]
> BTW could you please check if you can use `Utility.DESKTOP_ENVIRONMENT == 'gnome'` in place of the `is_gnome()` function.

Yeah that works. Committed the change

**Comment by slgobinath** [comment]
I've tested this on Fedora and works as expected. Once again thank you very much @filipkilibarda for your contribution

File: safeeyes/plugins/smartpause/plugin.py [hunk 1]
```diff
@@ -44,12 +46,54 @@
...
+
+
+def __gnome_wayland_idle_time():
+    """
+    Determine system idle time in seconds, specifically for gnome with wayland.
+    If there's a failure, return 0.
+    https://unix.stackexchange.com/a/492328/222290
+    """
+    try:
+        output = subprocess.check_output([
```
Comment (TPXP) [inline safeeyes/plugins/smartpause/plugin.py]: This could be made with the dbus library to be a little cleaner.

**Pull request merged**